src/ranged.cpp

```diff
@@ -776,8 +773,12 @@ void npc::pretend_fire( npc *source, int shots, item &gun )
     }
 }
 
-bool can_use_bipod( const map &m, const tripoint &pos )
+bool can_use_bipod( const Character &who, const map &m, const tripoint &pos )
```

Comment:
```suggestion
bool can_use_heavy_weapon( const Character &who, const map &m, const tripoint &pos )
```

the name `can_use_bipod` could be potentially confusing as it not only checks bipod but mechs as well.

Reply:
Aight, will need to change all calls to that function as well and not just the function name, so one sec then.